generic/proof-useropts.el

```diff
@@ -255,7 +255,9 @@ files which are out of date with respect to the loaded buffers!"
 
 (defcustom proof-prog-name-ask
   nil
-  "*If non-nil, query user which program to run for the inferior process."
+  "*If non-nil, query user which program to run for the inferior process.
+
+Note that ‘proof-prog-name’ is considered safe if this variable is non-nil."
```

Comment:
I think the prefered form of quoting in Emacs is `` `' ``, not `‘’`

Reply:
I don't know where I even got those quotes from.